microsoft-edge/webview2/releasenotes.md

```diff
@@ -14,18 +14,78 @@ keywords: IWebView2, IWebView2WebView, webview2, webview, win32 apps, win32, edg
 
 > [!NOTE]
-> Re-compile your app after updating the NuGet package.  
+> Ensure you re-compile your app after updating the NuGet package. The team recommends using the Canary channel when developing using the prerelease packages, and the evergreen runtime when using released packages. For more information, navigate to [Versioning][VersioningDoc].  
+ 
+## 1.0.773-prerelease  
+
+Release Date: January 25, 2021  
+
+[NuGet package][NuGetGallery1.0.773-prerelease] \| Microsoft Edge version 86.0.616.0 or newer  
+
+### General  
+
+> [!IMPORTANT]
+> WebView2 pre-release package 0.9.430 is deprecated, and is being removed from the upcoming release.  If your WebView app uses the package, we recommend updating to a newer package.  
+
+##### Features  
+
+*   Added [TrySuspend and Resume][ReferenceWin32Icorewebview210773PreReleaseTrySuspendResume] method to suspend and resume WebViews.  
+*   Added [SetVirtualHostNameToFolderMapping][ReferenceWin32Icorewebview210773PreReleaseSetVirtualHostNameToFolderMapping] method that maps a virtual host name to a directory path.  
+*   Added the [DefaultBackgroundColor][ReferenceWin32Icorewebview2controllerViewWebview210773PreReleaseDefaultBackgroundColor] property to set the background color and transparency.  \([\#414][GithubMicrosoftedgeWebviewfeedbackIssue414]\)  
+*   Added the [UserAgent][ReferenceWin32Icorewebview2experimentalsettings10773PreReleaseGetUserAgent] property to get or set the User Agent.\([\#122][GithubMicrosoftedgeWebviewfeedbackIssue549]\)
+*   Replaced the `CreateCookieWithCookie` method with the `CopyCookie` method.  
+*   Added visual hosting support using the [ICoreWebView2CompositionController][ReferenceWin32Icorewebview2controllerViewWebview210773CompositionController] interface, which is created using the new `CreateCoreWebView2CompositionController` method from `ICoreWebView2Environment3`. 
+
+    
+##### Bug fixes  
+
+*   Turned off the Microsoft Edge Shopping feature in WebView2.  
+*   Turned off the context menu in the PDF viewer when `AreDefaultContextMenusEnabled` is `false`.  \([\#605][GithubMicrosoftedgeWebviewfeedbackIssue605]\).  
+*   Fixed a bug that returned `E_NOINTERFACE` when querying `ICoreWebView2` for `ICoreWebView2Experimental`.  \([\#691][GithubMicrosoftedgeWebviewfeedbackIssue691]\).  
+*   Fixed a bug that allowed navigation with malformed URIs when `CoreWebView2NavigationStartingEventArgs.Cancel` was set to `false`.  \([\#400][GithubMicrosoftedgeWebviewfeedbackIssue400]\).  
+*   Fixed a bug that blocked `window.print()` on pop-up windows with event handlers attached to `NewWindowRequested` events.  \([\#409][GithubMicrosoftedgeWebviewfeedbackIssue409]\).  
+*   Fixed Dynamic DPI issue when moving apps between different monitors. \([\#58][GithubMicrosoftedgeWebviewfeedbackIssue549]\)  
```

Comment:
A definition for `GithubMicrosoftedgeWebviewfeedbackIssue58` is required too.
```suggestion
*   Fixed Dynamic DPI issue when moving apps between different monitors. \([\#58][GithubMicrosoftedgeWebviewfeedbackIssue58]\)  
```

Reply:
A definition for `GithubMicrosoftedgeWebviewfeedbackIssue58` is missing and `[ReferenceWin32IdlCreatecorewebview2environmentwithoptions09538` under 0.9.538 is missing a closing `]`